pyfr/integrators/dual/controllers.py

```diff
@@ -41,22 +45,60 @@ class DualNoneController(BaseDualController):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
-        self._niters = self.cfg.getint('solver-time-integrator', 'niters')
+        sect = 'solver-time-integrator'
+
+        self._innersteps = 0
+        self._maxniters = self.cfg.getint(sect, 'niters')
+
+        self._iatol = self.cfg.getfloat(sect, 'inner-atol')
+        self._irtol = self.cfg.getfloat(sect, 'inner-rtol')
+
+        self._chkevery = self.cfg.getint(sect, 'chk', default=5)
```

Comment:
Rename this variable/config option.  Also, do we want to force the user to specify a value rather than having a default?